implementations/rust/ockam/ockam_core/src/flow_control/access_control.rs

```diff
@@ -92,6 +92,16 @@ impl OutgoingAccessControl for FlowControlOutgoingAccessControl {
             }
         }
 
+        #[cfg(feature = "debugger")]
+        self.flow_controls.debug_addresses(
```

Comment:
I think it only makes sense to print debug info for current and next onward address, what is the point to debug the whole onward and return route? It's not even guaranteed that those addresses belong to our node

Reply:
Makes sense :+1: